recipes/liblsl/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(lsl CONFIG REQUIRED)
```

```suggestion
conan_basic_setup()

find_package(lsl CONFIG REQUIRED)
```

Sorry, wrong method.